app/src/main/java/com/zulip/android/ZulipApp.java

```diff
@@ -143,12 +171,127 @@ public int getAppVersion() {
         }
     }
 
+    public ObjectCache getObjectCache() {
```

Comment:
You're correct. I completely forgot to use the lazy getter for the `getDao` method. Sorry! Was in a hurry to get this in.